.bazelrc

```diff
@@ -10,4 +10,25 @@ build --nouse_workers_with_dexbuilder
 build:ci \
     --experimental_persistent_javac \
     --persistent_android_resource_processor \
-    --disk_cache=~/.cache/bazel/
\ No newline at end of file
+    --disk_cache=~/.cache/bazel/
+
+build --define=EXECUTOR=remote
```

Comment:
Ah yep we should definitely add that, but the error from the description is actually when a binary from the android sdk `aapt2` fails to execution because it's a macOS binary. I think theoretically that means we'd need something like this rbe_jdk but for the android sdk, wdyt?